dataset-spec/README.md

```diff
@@ -0,0 +1,62 @@
+
+*\* There is no standardized name for the concept we are describing here. Others called it: dataset series (ISO 19115), collection (CNES, NASA), dataset (JAXA), dataset series (ESA), product (JAXA).*
+
+## Core
+
+| Element         | Type                                  | Name                            | Description                                                  |
+| --------------- | ------------------------------------- | ------------------------------- | ------------------------------------------------------------ |
+| id              | string                                | Dataset ID (required)           | Identifier for the dataset that is unique across the provider. MUST follow the pattern ` ^[A-Za-z0-9_\-\/]+$ `. TODO: Allow slash? |
+| title           | string                                | Title                           | A short descriptive one-line title for the dataset.          |
+| description     | string                                | Description (required)          | Detailed multi-line description to fully explain the entity. [CommonMark 0.28](http://commonmark.org/) syntax MAY be used for rich text representation. |
+| keywords        | [string]                              | Keywords                        | List of keywords describing the dataset.                     |
+| version         | string                                | Dataset Version                 | Version of the dataset. [Semantic Versioning (SemVer)](https://semver.org/) SHOULD be followed. |
+| license         | string                                | Dataset License Name (required) | Dataset's license(s) as a [SPDX License identifier or expression](https://spdx.org/licenses/) or `proprietary` if the license is not on the SPDX license list. See `license_url` for more information. |
+| license_url     | string                                | Dataset License URL             | Dataset's license URL SHOULD be specified if `license` is set to `proprietary`. |
+| provider        | [Provider Object]                     | Data Provider                   | The organizations that created the content of the dataset.   |
+| host            | Host Object                           | Storage Provider                | The organization that hosts the dataset.                     |
+| spatial_extent  | [GeoJSON Object](http://geojson.org/) | Spatial extent (required)       | The spatial extent covered by the dataset as [GeoJSON](http://geojson.org/) object. |
+| temporal_extent | string                                | Temporal extent (required)      | Temporal extent covered by the dataset. Date/time intervals MUST be formatted according to ISO 8601. ToDo: Support open date ranges |
```

Comment:
It could be really good to try to get compatibility with WFS on the extent fields. They use:
```
      "extent": {
        "spatial": [ 7.01, 50.63, 7.22, 50.78 ],
        "temporal": [ "2010-02-15T12:34:56Z", "2018-03-18T12:11:00Z" ]
      },
```
If we want we can try to influence them to adopt our convention, but we should have good reasoning.

Reply:
* I'm perfectly fine with the additional level of hierarchy (i.e. putting them both in an object with the key extent).
* Spatial is fine, too.
* Temporal on the other hand is a problem. Open date ranges are missing. I already opened an issue on that: https://github.com/opengeospatial/WFS_FES/issues/155
* What is missing from your example is trs and crs. Do we need to support them?

Reply:
I just used 'example 4' from the WFS spec. It notes: 'Coordinate reference system information is not provided as the service provides geometries only in the default system (WGS84 longitude/latitude)'. So seems like we could just say for dataset we require default.

Reply:
Okay, than that's fine. Curious what the WFS crew is coming up with for the other issues mentioned. Will change that in the dataset spec. Temporal extent is still a string for now and need to find out how they define 3D bboxes (= incl. the z-axis).

Reply:
Should be mostly WFS compatible now.